staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/status.go

```diff
@@ -35,14 +35,19 @@ func ErrorToAPIStatus(err error) *metav1.Status {
 	switch t := err.(type) {
 	case statusError:
 		status := t.Status()
-		if len(status.Status) == 0 {
```

Comment:
keep this in place, defaulting unspecified to failure seems ok

Reply:
ack will add it back